src/signals/settings/users/containers/Overview/__tests__/Overview.test.js

```diff
@@ -440,4 +422,176 @@ describe('signals/settings/users/containers/Overview', () => {
+
+    await wait();
+
+    expect(getByTestId('userActiveSelect')).toBeInTheDocument();
+  });
+
+  it('should select the right option when user active (status) filter changes', async () => {
+    const { getByTestId } = render(usersOverviewWithAppContext());
+
+    await wait();
+
+    const filterByUserActiveSelect = getByTestId('userActiveSelect');
+
+    // check if the default value has been set
+    expect(filterByUserActiveSelect.value).toBe('*');
+
+    const filterValue = 'true';
+
+    act(() => {
+      fireEvent.change(filterByUserActiveSelect, { target: { value: filterValue } });
+    });
+
+    expect(dispatch).toHaveBeenCalledTimes(1);
+    expect(dispatch).toHaveBeenCalledWith(setUserFilters({ is_active: filterValue }));
+
+    expect(filterByUserActiveSelect.value).toBe(filterValue);
+  });
+
+  it('should keep the state of all filters in context', async () => {
+    const { getByTestId } = render(usersOverviewWithAppContext());
+
+    await wait();
+
+    const filterByUserActiveSelect = getByTestId('userActiveSelect');
+    const filterByRoleSelect = getByTestId('roleSelect');
+
+    // check if the default values have been set
+    expect(filterByUserActiveSelect.value).toBe('*');
+    expect(filterByRoleSelect.value).toBe('*');
+
+    const userActiveFilterValue = 'true';
+    const roleFilterValue = 'Regievoerder';
+
+    act(() => {
+      fireEvent.change(filterByUserActiveSelect, { target: { value: userActiveFilterValue } });
+      fireEvent.change(filterByRoleSelect, { target: { value: roleFilterValue } });
+    });
+
+    expect(dispatch).toHaveBeenCalledTimes(2);
+
+    expect(dispatch).toHaveBeenCalledWith(setUserFilters({ is_active: userActiveFilterValue }));
+    expect(dispatch).toHaveBeenCalledWith(setUserFilters({ role: roleFilterValue }));
+
+    expect(filterByUserActiveSelect.value).toBe(userActiveFilterValue);
+    expect(filterByRoleSelect.value).toBe(roleFilterValue);
+  });
+
+  it('dispatches the correct actions on successful getSeason fetch request', async () => {
+    const { getByTestId } = render(usersOverviewWithAppContext());
+
+    await wait();
+
+    fetch.mockResponses(
+      [JSON.stringify(usersFiltersDisabledJSON), { status: 200 }],
+      [JSON.stringify(usersFiltersActiveJSON), { status: 200 }],
+      [JSON.stringify(usersFilterInactiveJSON), { status: 200 }],
+      [JSON.stringify(usersFiltersActiveAndRoleJSON), { status: 200 }],
+      [JSON.stringify(usersFiltersInactiveAndRoleJSON), { status: 200 }],
+    );
+
+    const filterByUserActiveSelect = getByTestId('userActiveSelect');
+    const filterByRoleSelect = getByTestId('roleSelect');
+    const apiPrefix = 'https://acc.api.data.amsterdam.nl/signals/v1/private/users?page=1&page_size';
+
+    // filters: disabled
+    expect(filterByUserActiveSelect.value).toBe('*');
+    expect(filterByRoleSelect.value).toBe('*');
+
+    const filtersDisabled = await fetch(apiPrefix);
```

Comment:
Why would you need to `fetch` here if you can just use the JSON form the fixture?

Reply:
Also not needed anymore in the new implementation.